docs/index.rst

```diff
@@ -94,9 +94,75 @@ These arguments relate to connections created from the database.
 
 connection_cache_size
   integer (default 10000)
+
+  Target size, in number of objects, of each connection's object
+  cache.
+
+connection_cache_size_bytes
+  integer (default 0)
+
+  Target size, in total estimated size for objects, of each
+  connection's object cache.  0 means no limit.
+
+  A suffix of KB, MB, or GB may be used to provide units.
+
+connection_historical_cache_size
+  integer (default 1000)
+
+  Target size, in number of objects, of each historical connection's
+  object cache.
+
+connection_historical_cache_size_bytes
+  integer (default 0)
+
+  Target size, in total estimated size of objects, of each historical
+  connection's object cache.
+
+  A suffix of KB, MB, or GB may be used to provide units.
+
+connection_historical_pool_size
+  integer (default 3)
+
+  The expected maximum total number of historical connections
+  simultaneously open.
+
+connection_historical_timeout
+  integer (default 300)
+
+  Maximum age of inactive historical connections When a historical
```

Comment:
Gaaaa. Most of this is of the form:

phrase (not sentence) saying what it is.

Optional sentences saying. This doesn't quite fit the structure. I'll need to fiddle a bit nore...